features/currency/managing_currency/adding_currency.feature

```diff
@@ -0,0 +1,17 @@
+@currency
+Feature: Adding a new currency
+  In order to receive payments in different currencies
+  As an Administrator
+  I want to add a new currency to the registry
+
+  Background:
+    Given I am logged in as an administrator
+
+  @ui
+  Scenario: Adding a new currency
+    Given I want to create a new currency
```

Comment:
Only we we are actually creating things, not when they exist, but we only add them as available in our system ;) Look at countries for instance.